client/components/operations/label.js

```diff
@@ -143,6 +143,13 @@ export class OperationListViewLabel extends LabelComponent {
     }
 
     render() {
+        if (window.innerWidth < 768) {
```

Comment:
I am really not a big fan of mixing dynamic rendering and styling rules. Could you use only CSS classes to the same effect, please?